spec/features/users/sign_in_spec.rb

```diff
@@ -92,10 +92,22 @@
   end
 
   context 'signed out' do
+    it 'keeps the user on the current page after session times out', js: true do
+      allow(Devise).to receive(:timeout_in).and_return(0)
+
+      visit new_user_registration_path
+      fill_in 'Email', with: 'teststuff'
+
+      find_link(t('forms.buttons.continue')).trigger('click')
+
+      expect(find_field('Email').value).to be_blank
+      expect(current_path).to eq new_user_registration_path
+    end
+
```

Comment:
What do you think about combining this test with the one below? Tests with JS enabled are slow, and since we're testing the same feature, I think it's fine to combine the two expectations: 1) that the modal is displayed, and 2) that clicking continue leaves you on the current page. I propose something like this:
```ruby
it 'displays session expired modal with link to current page', js: true do
  allow(Devise).to receive(:timeout_in).and_return(0)

  visit new_user_registration_path

  expect(page).to have_css('#session-expired-msg')

  find_link(t('forms.buttons.continue')).trigger('click')

  expect(current_path).to eq new_user_registration_path
end
```